tensorflow-framework/src/main/java/org/tensorflow/framework/losses/impl/LossesImpl.java

```diff
@@ -0,0 +1,303 @@
+package org.tensorflow.framework.losses.impl;
+
+import org.tensorflow.DataType;
+import org.tensorflow.Operand;
+import org.tensorflow.framework.losses.Reduction;
+import org.tensorflow.ndarray.Shape;
+import org.tensorflow.op.Ops;
+import org.tensorflow.op.core.ReduceSum;
+import org.tensorflow.op.core.Squeeze;
+import org.tensorflow.types.TInt32;
+import org.tensorflow.types.family.TNumber;
+
+import java.util.Collections;
+
```

Comment:
Class should have Javadoc description, no?

Reply:
I have changed the class name to `LossesHelper`.

I don't understand your comment on Class JavaDoc. This is what I have in my copy.
```
/**
 * These are helper methods for Losses and Metrics and will be module private when Java modularity is applied to
 * TensorFlow Java. These methods should not be used outside of the losses and metrics packages.
 */
```

The basic comment was put in a while a ago, and I just updated it to mention `metrics`.

Reply:
Rendering issue I think. Looks good, thanks.